packages/bsky/src/data-plane/server/routes/labels.ts

```diff
@@ -14,10 +14,14 @@ export default (db: Database): Partial<ServiceImpl<typeof Service>> => ({
     if (subjects.length === 0 || issuers.length === 0) {
       return { labels: [] }
     }
+
     const res: LabelRow[] = await db.db
       .selectFrom('label')
       .where('uri', 'in', subjects)
       .where('src', 'in', issuers)
+      .where((qb) =>
+        qb.where('exp', 'is', null).orWhere(sql`exp::timestamp > now()`),
+      )
```

Comment:
Should we omit not only expired but also negated labels here?  The negation and expiration feel like they are part of the same kind of logic to me.  Though most importantly is really just that this has the same behavior as the production dataplane 👍